README.md

```diff
@@ -66,6 +66,9 @@ fill them in
         $ python api/cfc_webapp.py
 
    You may need to install some of the python dependencies from above
+   Amongst the dependencies include:
+   -cherrypy
```

Comment:
Ah, make sense. I should really move that import inside the "if ssl" code block.
I'll merge this change now - can you file an issue for the code fix so that I can get to it later?